src/itemChoice/index.tsx

```diff
@@ -32,6 +33,7 @@ export interface ItemChoiceProps {
   readonly onMouseDown?: (event: React.MouseEvent<HTMLElement>) => void
   readonly onDoneAnimationEnd?: () => void
   readonly ariaLabel?: string
+  readonly ariaDescribedby?: string
```

Comment:
You already added the prop in A11yProps

Reply:
Oh yeah, forgot to clean it. Thanks for spotting it.